uber/templates/registration/new.html

```diff
@@ -144,6 +144,8 @@
         <td>
             {% if attendee.paid == HAS_PAID %}
                 paid
```

Comment:
Since we don't check in unclaimed group badges on this page anymore, I didn't want the forms to be active for ``PAID_BY_GROUP`` badges.